File: 10_기타알고리즘/01_소수판별_기본.py
def isPrimeNum(x) :
    for i in range(2, x) :
        # 2 ~ (x-1) 중 나누어 떨어지는 수가 있으면 소수가 아님
        if(x % i == 0) :
            return False
        return True

print(isPrimeNum(4))
print(isPrimeNum(7))

File: 10_기타알고리즘/01_소수판별_기본.cpp
#include <iostream>

using namespace std;

bool isPrimeNum(int x) {
    for (int i = 2; i <= x-1; i++)
    {
        if(x % i == 0) {
            return false;
        }
    }
    return true;
}

int main(void){

    cout << boolalpha;
    cout << isPrimeNum(4) << endl;
    cout << isPrimeNum(7) << endl;

    return 0;
}